utils.cpp

```diff
@@ -102,4 +99,13 @@ void actualizarDatos(ofstream &archivo, Contacto* pAgenda, int &cdadContactos) {
         archivo << pAgenda[i].numero << endl;
         i++;
     }
-}
\ No newline at end of file
+}
+
+/* Ese es el arreglo que estoy haciendo blablabla
+ *
+ *
+ *
+ *
+ *
+ *
+ */
```

Comment:
Me parece mejor hacer algo asi
```suggestion
void arreglo() { 
// Aca va ese codigo
}
```